src/src/exe.zig

```diff
@@ -0,0 +1,191 @@
+const std = @import("std");
```

Comment:
note for reviewers: ignore this file, it's my testing file to assert native behaviors, it'll get deleted in the future